lang/vendor/rapidez/nl/frontend.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+return [
+    'cart' => [
+        'add'    => 'toegevoegd aan de winkelwagen.',
+        'remove' => 'verwijderd van de winkelwagen.',
+        'coupon' => [
+            'applied' => 'Kortingscode succesvol toegepast!',
+        ],
+    ],
+    'errors' => [
+        'wrong'           => 'Er ging iets fout, probeer het opnieuw.',
+        'session_expired' => 'Je sessie is verlopen, log opnieuw in.',
+        'cart_expired'    => 'Je winkelwagen is verlopen, voeg de producten opnieuw toe.',
+    ],
+    'checkout' => [
+        'no_shipping_method' => 'Geen verzendmethode geselecteerd.',
+        'no_payment_method'  => 'Geen betaalmethode geselecteerd.',
+    ],
+    'account' => [
+        'password_mismatch' => 'De wachtwoorden komen niet overeen.',
+        'email_password'    => 'Je hebt geen email of wachtwoord opgegeven.',
+        'email'             => 'Een email adres is verplicht.',
+    ],
+
+    'asc'       => 'Oplopend',
+    'desc'      => 'Aflopend',
+    'relevance' => 'Relevantie',
+    'newest'    => 'Nieuwste',
+    'all'       => 'Alles',
+
+    // 'sorting' => [
```

Comment:
maybe add
    'sorting' => [
        'price' => [
            'asc' => 'Prijs oplopend',
            'desc' => 'Prijs aflopend',
        ],
        'name' => [
            'asc' => 'Naam A-Z',
            'desc' => 'Naam Z-A',
        ],
    ],
    
    since they dont get translated this way